gangplank/cmd/gangplank/pod.go

```diff
@@ -95,11 +95,18 @@ func runPod(c *cobra.Command, args []string) {
 	if cosaViaPodman {
 		if cosaPodmanRemote != "" {
 			os.Setenv(podmanRemoteEnvVar, cosaPodmanRemote)
-			minioSshRemoteHost = containerHost()
-			if strings.Contains(minioSshRemoteHost, "@") {
-				parts := strings.Split(minioSshRemoteHost, "@")
-				minioSshRemoteHost = parts[1]
-				minioSshRemoteUser = parts[0]
+			if minioCfgFile != "" {
```

Comment:
```suggestion
			if minioCfgFile == "" {
```

Don't we need to flip the logic here. We only want to define our ssh port forwarding vars IF there was no `minioCfgFile` provided.

I was originally planning to rebase my PR on this one so I resolved the merge conflict yesterday like this:

```diff
diff --git a/gangplank/cmd/gangplank/pod.go b/gangplank/cmd/gangplank/pod.go
index 40c5234f..9a61958e 100644
--- a/gangplank/cmd/gangplank/pod.go
+++ b/gangplank/cmd/gangplank/pod.go
@@ -95,22 +95,17 @@ func runPod(c *cobra.Command, args []string) {
        if cosaViaPodman {
                if cosaPodmanRemote != "" {
                        os.Setenv(podmanRemoteEnvVar, cosaPodmanRemote)
-                       if minioCfgFile != "" {
-                               minioSshRemoteHost = containerHost()
-                               if strings.Contains(minioSshRemoteHost, "@") {
-                                       parts := strings.Split(minioSshRemoteHost, "@")
-                                       minioSshRemoteHost = parts[1]
-                                       minioSshRemoteUser = parts[0]
-                               }
-                               log.WithFields(log.Fields{
-                                       "remote user":     minioSshRemoteUser,
-                                       "ssh foward host": minioSshRemoteHost,
-                                       "container host":  cosaPodmanRemote,
-                               }).Info("Minio will be forwarded to remote host")
-                       }
-                       if cosaPodmanRemoteSshKey != "" {
-                               os.Setenv(podmanSshKeyEnvVar, cosaPodmanRemoteSshKey)
-                               minioSshRemoteKey = cosaPodmanRemoteSshKey
+               }
+               if cosaPodmanRemoteSshKey != "" {
+                       os.Setenv(podmanSshKeyEnvVar, cosaPodmanRemoteSshKey)
+               }
+               if cosaPodmanRemote != "" && minioCfgFile == "" {
+                       minioSshRemoteHost = containerHost()
+                       minioSshRemoteKey = cosaPodmanRemoteSshKey
+                       if strings.Contains(minioSshRemoteHost, "@") {
+                               parts := strings.Split(minioSshRemoteHost, "@")
+                               minioSshRemoteHost = parts[1]
+                               minioSshRemoteUser = parts[0]
                        }
                        log.WithFields(log.Fields{
                                "remote user":     minioSshRemoteUser,
```

Reply:
I've corrected the `minioCfgFile == ""` logic  and retested.